src/RabbitMQPublisher.php

```diff
@@ -33,7 +33,7 @@ class RabbitMQPublisher implements Publisher
     private $additionalPropertiesResolver;
 
     public function __construct(
-        MessageInEnvelopSerializer $messageSerializer,
+        MessageInEnvelopeSerializer $messageSerializer,
```

Comment:
Same here. This may be considered a BC break. If bumping major version then this is fine.